### ISSUE TITLE
`--migrations` to specify output directory for migrations

### ISSUE THREAD

**Issue by janpio** [prisma/migrate#361]
With `--schema` we can now specify the file path of where the Prisma Schema is located, instead of the default `./prisma/schema.prisma`. This has the unfortunate side effect, that migrations are also output next to that file - which might not be what you want.

We want to have a `--migrations` parameter that takes the path to a _folder_ where all the migration related content is output. The default path for that parameter, if it is not supplied, thus is `./prisma/migrations`.

**Comment by maoosi** [comment]
Super keen to see this feature come to life. I use Prisma Migrate across multiple stages, so the ability to specify a different migration folder for each Stage would be quite helpful.

**Comment by rightroad** [comment]
it would be better if the migrations output could be set via package.json as follows:

 "prisma": {
    "schema": "src/schema.prisma",
    "migrateOutput" : "src/migrations/"
  }

**Comment by maoosi** [comment]
I have to somewhat disagree @rightroad, as solely relying on `package.json` to specify the migration output would exclude a certain number of use cases, like the one I just described around managing multiple stages from the same project. Adding an option to the CLI would allow to easily switch configs at runtime.

That being said, it would make sense to have both options available and align with what's already in place for the Prisma schema file location: https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema#prisma-schema-file-location

**Comment by rightroad** [comment]
Yeah thank you for your reply @maoosi, having both options would be great! I think .env file also needs that change (or use .env on src folder) but that shouldn't be a big deal.
